qds_sdk/commands.py

```diff
@@ -254,6 +254,98 @@ def parse(cls, args):
         v["command_type"] = "HiveCommand"
         return v
 
+class SparkCommand(Command):
+	
+	#{"program":prog,"language":"python","command_type":"SparkCommand","label":"praveen-spark"} 
+    usage = ("sparkcmd <submit|run> [options]")
+    allowedlandlist = ["python", "scala"]
+    optparser = GentleOptionParser(usage=usage)
+    optparser.add_option("--program", dest="program", help="program code")
+    
+    optparser.add_option("--cmdline", dest="commandline", help="command line for Spark")
+        
+    optparser.add_option("-f", "--script_location", dest="script_location",
+                         help="Path where spark program to run is stored. Can be S3 URI or local file path")
+
+    optparser.add_option("--macros", dest="macros",
+                         help="expressions to expand macros used in query")
+	
+    optparser.add_option("--tags", dest="tags",
+                         help="comma-separated list of tags to be associated with the query ( e.g., tag1 tag1,tag2 )")
+
+    optparser.add_option("--sample_size", dest="sample_size",
+                         help="size of sample in bytes on which to run query")
+
+    optparser.add_option("--cluster-label", dest="label", help="the label of the cluster to run the command on")
+	
+    optparser.add_option("--language", dest="language", help="the language used in the spark program, supported languages are scala and python")
```

Comment:
Can we use `choices` here?
https://docs.python.org/2/library/optparse.html#optparse.Option.choices